### ISSUE TITLE
Administration of your own instance

### ISSUE THREAD

**Issue by Tealk** [linkwarden/linkwarden#300]
Hello,

even after a bit of searching through the repo and the documentation, I have not found anything about how to administer an instance. Is there currently no way to do this?

**Comment by jontstaz** [comment]
Yes it's possible, you just use the Docker image and spin up a container. I've been running Linkwarden on my own server for some months now.

**Comment by Tealk** [comment]
> Yes it's possible, you just use the Docker image and spin up a container

And how do I manage this?
I would have expected something like an /admin route that opens an admin interface where I can e.g. delete users or do other administrative tasks.

**Comment by patrickasmus** [comment]
I agree with the question. 
Is there no administration option for Docker users?

**Comment by daniel31x13** [comment]
Related to #217, this feature is already part of the todo’s.

**Comment by patrickasmus** [comment]
I saw it two minutes later too. Sorry for acting too quickly
